src/setup.erl

```diff
@@ -858,7 +871,7 @@ run_setup(Parent, Args) ->
     end.
 
 run_setup_(Parent, _Args) ->
-    Res = rpc:multicall(?MODULE, verify_directories, []),
+    Res = maybe_verify_directories(),
```

Comment:
Yeah, I believe we actually had use for it in a project you and I worked together, but I'm willing to drop it and wait for the requirement to resurface. :)